setup.py

```diff
@@ -0,0 +1,10 @@
+# type: ignore
+# Used for pre-commit since it expects a setup.py in repo root
+# for actual setup.py see semgrep/setup.py
+from setuptools import setup
+
+setup(
+    name="pre_commit_dummy_package",
```

Comment:
Should we instead use something like `semgrep_pre_commit_package`? That way if we ever see this string anywhere it'll give us a hint as to where it's coming from. We may also want to grab this PyPI package name just so there's no funny business ever going on with that.

Reply:
Went with your suggestion and parked the name